src/webrtc/call.ts

```diff
@@ -2038,53 +2111,121 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
         return;
     }
 
-    public subscribeToFocus(): void {
-        if (!this.remoteSDPStreamMetadata) return;
+    /**
+     * Send an m.call.track_subscription event to the focus. The method is
+     * throttled to avoid spamming the focus with events when scrolling or
+     * resizing the window.
+     * The m.call.track_subscription lets the focus know about the visibility
+     * and size of different tracks.
+     * @param force - whether or not to force the request to be sent immediately
+     */
+    public subscribeToFocus(force = false): void {
+        // TODO: Can we throttle this better?
```

Comment:
lodash.throttle / debounce are your friends

Reply:
Ah, ok - ftr I'd vote for adding it: it will only pull in the bits you use so it's not expensive.